packages/electron-builder/package.json

```diff
@@ -66,6 +66,7 @@
     "parse-color": "^1.0.0",
     "plist": "^2.0.1",
     "progress": "^1.1.8",
+    "s3": "^4.4.0",
```

Comment:
Dependency should be added to the root package.json as well (for tests). But I am going to any case move S3 publisher to a separate package (to reduce dependencies if user doesn't use S3 publisher), so, I will do it.